sigmf-spec.md

```diff
@@ -27,24 +27,18 @@ Copyright of contributions to SigMF are retained by their original authors. All
   - [Introduction](#introduction)
   - [Conventions Used in this Document](#conventions-used-in-this-document)
   - [Specification](#specification)
-    - [Files](#files)
-    - [SigMF Archives](#sigmf-archives)
-    - [Dataset Format](#dataset-format)
-    - [Metadata Format](#metadata-format)
+    - [SigMF File Types](#sigmf-file-types)
+    - [SigMF Dataset Format](#sigmf-dataset-format)
+    - [SigMF Metadata Format](#sigmf-metadata-format)
       - [Datatypes](#datatypes)
       - [Namespaces](#namespaces)
         - [Extension Namespaces](#extension-namespaces)
       - [Global Object](#global-object)
-        - [The `geolocation` Field](#the-geolocation-field)
-        - [The `extensions` Field](#the-extensions-field)
-        - [The `collection` Field](#the-collection-field)
```

Comment:
i dont think this level of detail is helpful or useful in the TOC and to update this to be consistent would result in a ton of entries here.